codegeex.el

```diff
@@ -31,56 +32,48 @@
 (require 'url)
 (require 'json)
 (require 'uuidgen)
+(require 'codegeex-completion)
+(require 'codegeex-overlay)
 
-(defvar codegeex-endpoint "https://tianqi.aminer.cn/api/v2/" "the endpoint of CodeGeeX API")
-(defvar codegeex-apikey "68cf004321e94b47a91c2e45a8109852" "API key obtained from CodeGeeX website")
-(defvar codegeex-apisecret "e82b86a16f9d471ab215f653060310e3" "API secret obtained from CodeGeeX website")
-(defvar codegeex-temperature 0.2 "temperature for completion by CodeGeeX")
-(defvar codegeex-top_p 0.95 "top_p for completion by CodeGeeX")
-(defvar codegeex-top_k 0 "top_k for completion by CodeGeeX")
-(defvar codegeex-extinfo `((sid . ,(uuidgen-4))
```

Comment:
This is the only usage of uuid package. Maybe put the function in the package to avoid dependencies would be interesting

Reply:
Let's remove it right now then and we can rethink about this in the future, when we will get a correct API documentation (if we have one)